src/ADAL.PCL.Desktop/WebUI.cs

```diff
@@ -44,11 +44,20 @@ internal abstract class WebUI : IWebUI
         {
             AuthorizationResult authorizationResult = null;
 
+            string requestId = Telemetry.GetInstance().RegisterNewRequest();
+            Telemetry.GetInstance().StartEvent(requestId, "ui_time");
+
+            UIEvent UiEvent = new UIEvent();
+            DateTime UiStartTime = DateTime.Now;
             var sendAuthorizeRequest = new Action(
                 delegate
                 {
                     authorizationResult = this.Authenticate(authorizationUri, redirectUri);
                 });
+            DateTime stopTime = DateTime.Now;
+            UiEvent.UiTime = stopTime.Subtract(UiStartTime);
+            UiEvent.SetEvent(EventConstants.UiTime, UiEvent.UiTime);
+            Telemetry.GetInstance().StopEvent(requestId, UiEvent, "ui_event");
```

Comment:
>ui_event [](start = 67, length = 8)

This one is troubling, how is your unit test not catching this? start event (ui_time) and stop event(ui_event) names are different.